print_cmd.cpp

```diff
@@ -7,11 +7,15 @@ using namespace std: use names for objects and variables from the standard libra
 */
 
 int main() {
-  cout << "Hello GarbyX \n\n";
+  cout << "Hello GarbyX" << endl;
```

Comment:
The endline manipulator serves same use as newline escape character and is well implemented.